### PR TITLE
Exercicio4

### PR DESCRIPTION
**Pull request by thiagopinheirolima**

### CODE REVIEW

File: Exercicio 4/MainPage.xaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="auto"/>
+            <ColumnDefinition Width="*"/>
+        </Grid.ColumnDefinitions>
+
+        <StackPanel VerticalAlignment="Center"  Grid.Column="0">
```
Comment (soutolucas) [inline Exercicio 4/MainPage.xaml]: Is redundant set Grid.Column="0" on first column.

File: Exercicio 4/MainPage.xaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            <TextBlock x:Name="NoContentText" HorizontalAlignment="Center" Visibility="Collapsed" FontSize="50">
+                No Content Available
+            </TextBlock>
+
+            <ProgressRing x:Name="ProgressRing" HorizontalAlignment="Center" IsActive="True" Foreground="Black" Width="100" Height="100" Visibility="Collapsed"/>
+        </StackPanel>
+        
+        
+
+        <VisualStateManager.VisualStateGroups>
```
Comment (soutolucas) [inline Exercicio 4/MainPage.xaml]: Could you please move "VisualStateManager" to the top of the page?

File: Exercicio 4/MainPage.xaml (hunk 1 above)
Comment (thiagopinheirolima) [reply Exercicio 4/MainPage.xaml]: Done

File: Exercicio 4/MainPage.xaml (hunk 2 above)
Comment (thiagopinheirolima) [reply Exercicio 4/MainPage.xaml]: Done

**Pull request merged**